test/backup/backup_test.go

```diff
@@ -960,7 +962,39 @@ func TestBackups_integration(t *testing.T) {
 				relevant = append(relevant, backup)
 			}
 		}
-		require.Len(t, relevant, 3, "wrong number of backups")
+
+		// There may be other backups created in other tests;
+		require.True(t, len(relevant) >= 3, "wrong number of backups")
+	})
+
+	t.Run("list backups with ascending order", func(t *testing.T) {
+		testsuit.AtLeastWeaviateVersion(t, client, "1.32.2", "List backups sorting is only supported from 1.33.2")
+
+		testsuit.CreateTestSchemaAndData(t, client)
+		defer testsuit.CleanUpWeaviate(t, client)
+
+		assertAllPizzasExist(t, client)
```

Comment:
nit: I don't think this assertion is necessary here -- the test only tries to assert that backups are listed in a specific order.